### PR TITLE
Fixed ChangeBgY_ScreenOff‘s signature.

### PR DESCRIPTION
**Pull request by gAlfonso-bit**
It’s the only function of the “ChangeBy” functions that uses a u32 for value when it should be s32 like the rest.

## **Discord contact info**

AlfonsoB#1500

### CODE REVIEW

File: gflib/bg.c [hunk 1]
```diff
@@ -695,7 +696,7 @@ s32 ChangeBgY(u8 bg, s32 value, u8 op)
     return sGpuBgConfigs2[bg].bg_y;
 }
 
-s32 ChangeBgY_ScreenOff(u8 bg, u32 value, u8 op)
+s32 ChangeBgY_ScreenOff(u8 bg, s32 value, u8 op)
```
Comment (huderlem) [inline gflib/bg.c:699]: Maybe they should all be `u32`.  How do you know which is right?

File: gflib/bg.c [hunk 2]
```diff
@@ -252,17 +252,18 @@ static void SetBgAffineInternal(u8 bg, s32 srcCenterX, s32 srcCenterY, s16 dispC
 
     switch (sGpuBgConfigs.bgVisibilityAndMode & 0x7)
     {
+
```
Comment (huderlem) [inline gflib/bg.c:255]: Why did you move the `default` and `0` cases to the top of the `switch`, in addition to adding an extra newline?

File: gflib/bg.c [hunk 3]
```diff
@@ -280,10 +281,10 @@ static void SetBgAffineInternal(u8 bg, s32 srcCenterX, s32 srcCenterY, s16 dispC
     SetGpuReg(REG_OFFSET_BG2PC, dest.pc);
     SetGpuReg(REG_OFFSET_BG2PD, dest.pd);
     SetGpuReg(REG_OFFSET_BG2PA, dest.pa);
-    SetGpuReg(REG_OFFSET_BG2X_L, (s16)(dest.dx));
```
Comment (huderlem) [inline gflib/bg.c:283]: Does this match without the explicit casts?

File: gflib/bg.c (hunk 1 above)
Comment (gAlfonso-bit) [reply gflib/bg.c:698]: Others are s32 and the values being operated on are signed. Plus it makes no sense why a delta parameter should be unsigned
